packages/react-components/react-motion-components-preview/library/src/choreography/Stagger/utils/react-children-utils.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+/**
+ * Flattens ReactNode (including Fragments) to an array of valid ReactElements,
+ * filtering out strings, numbers, null, etc.
+ */
+export const toElementArray = (children: React.ReactNode): React.ReactElement[] =>
```

Comment:
I don't see anything else being used yet, except `toElementArray`. Let's keep only it.

Also suggest to rewrite it to emit a warning on invalid `children`. Something like:

```
export const toElementArray = (children: React.ReactNode): React.ReactElement[] => {
  const result: React.ReactElement[] = [];

  React.Children.forEach(children, child => {
    if (React.isValidElement(child)) {
      result.push(child);
    }

    if (process.env.NODE_ENV !== 'production') {
      // eslint-disable-next-line no-console
      console.error(['toElementArray: Ignoring non-ReactElement child:', child]);
    }
  });

  return result;
};
```

Otherwise people won't know that something goes wrong